modules/search/instant-search/components/search-result.jsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { sprintf, _n } from '@wordpress/i18n';
+import { h, Component } from 'preact';
+import strip from 'strip';
+
+class SearchResult extends Component {
+	render() {
+		return (
+			<div className="jetpack-instant-search__result">
+				<a
+					href={ `//${ this.props.result.fields[ 'permalink.url.raw' ] }` }
+					target="_blank"
+					rel="noopener noreferrer"
+					className="jetpack-instant-search__result-title"
+				>
+					{ strip( this.props.result.fields.title_html ) || 'Unknown Title' }
+				</a>{' '}
+				<div className="jetpack-instant-search__result-author-and-date">
+					{ strip( this.props.result.fields.author ) }{' '}
+					<span className="jetpack-instant-search__result-date">
+						{ strip( this.props.result.fields.date ).split( ' ' )[ 0 ] }
+					</span>
+				</div>
+				<div className="jetpack-instant-search__result-excerpt">
+					{ strip( this.props.result.fields.excerpt_html ) }
```

Comment:
Same encoding issues as above:

![image](https://user-images.githubusercontent.com/426388/63436806-a3a95e80-c429-11e9-8a64-e2a79e6c53b2.png)

Reply:
This is now being tracked in #13291.